External/FEXCore/Source/Interface/IR/IR.json

```diff
@@ -1109,31 +1100,46 @@
       ]
     },
 
-    "UCVTF": {
+    "FCVTFromGPR_U": {
+      "Desc": "Scalar op: Converts unsigned GPR to Scalar float",
       "HasDest": true,
       "SSAArgs": "1",
       "Args": [
         "uint8_t", "ElementSize"
       ]
     },
 
-    "SCVTF": {
+    "FCVTFromGPR_S": {
+      "Desc": "Scalar op: Converts signed GPR to Scalar float",
       "HasDest": true,
       "SSAArgs": "1",
       "Args": [
         "uint8_t", "ElementSize"
       ]
     },
 
-    "VUCVTF": {
+    "FCVTFloatToFloat": {
```

Comment:
This is inconsistent in multiple ways.

 1. It really should be renamed to `FCVT_FToF` for consistency with `VCVT_UToF` and friends. Or maybe those should be renamed to `VCVT_UnsignedToFloat` for more clarity. 
 2. Scalar has one operation that switches between multiple sizes. Vector has two operations `VCVT_FNarrow`/`VCVT_FExtend`.
The naming of these be uniform.